state/logs.go

```diff
@@ -66,6 +66,9 @@ var logIndexes = [][]string{
 	// the same time have a consistent ordering.
 	{"e", "t", "_id"},
 	{"e", "n"},
+
+	// This index is used for pruning logs by model and time.
+	{"e", "t"},
```

Comment:
Oh and for posterity: I can't see a way to use hints when removing. You can with find/queries, but not with RemoveAll. There's a $hint operator, but Mongo complained when I tried to use that in a remove operation.